src/phasmophobia-data/data.js

```diff
@@ -53,45 +131,53 @@ var data = {
 			"primaryEvidences": [
 				"Ghost Orbs",
 				"Fingerprints",
-				"D.O.T.S. Projector",
+				"D.O.T.S. Projector"
 			],
 			"secondaryEvidences": [
 				"Targets and hunts one player until killed (If the player is not in the building, it does a regular hunt)",
 				"The Crucifixs effective radius is 5m instead of 3m",
-				"Only choosen player can finish the 'Repel a ghost while it is chasing' objective"
+				"Only choosen player can finish the 'Repel a ghost while it is chasing' objective",
+				"Preferes singing ghost events over other events",
+				"Low chance for a unique sound (distorted scream) over the parabolic microphone"
 			]
 		},
 		{
 			"name": "Jinn",
 			"primaryEvidences": [
 				"Freezing Temperature",
 				"EMF Level 5",
-				"Fingerprints",
-			],
-			"secondaryEvidences": [
-				"During a hunt it travels faster when chasing a target until it comes close",
+				"Fingerprints"
+      ],
+      "secondaryEvidences": [
```

Comment:
Intendation is messed up a little.

Reply:
Don't know what is going on here. They are normal tabs and look well in atom. maybe just a github visual glitch?

![Screenshot 2022-01-05 at 19 09 04](https://user-images.githubusercontent.com/925714/148267278-85230117-67ec-497c-b921-841e14293c25.png)

Reply:
Hm, strange indeed. I will check on my end as well and fix if there is anything to fix.